extensions/language-negotiation-3.2.md

```diff
@@ -0,0 +1,73 @@
+---
+title: IRCv3.3 Language Negotiation
+layout: spec
+copyrights:
+  -
+    name: "Peter Powell"
+    period: "2014-2015"
+    email: "petpow@saberuk.com"
+---
+
+## About
+
+This specification allows clients to request that messages from the server are sent in a specified language. It does not define any translation behaviour for `NOTICE` or `PRIVMSG` except in the case of network pseudoclients such as services.
+
+### Capability
+
+Available languages are sent using the `languages` capability. The value of the capability is a comma delimited list of tokens. The first token is an unsigned 8-bit integer which specifies the number of fallback languages which can be requested. All tokens after that are a list of BCP 47 language codes that the server supports in the order they are used by default.
```

Comment:
Should that integer be the maximum number of *fallback* language codes that can be requested, or the maximum number of language codes that can be requested in total?

*Fallback* implies that the first requested language doesn't count towards the limit (and if that's the case, example 2 needs to have another language added to the client request for it to be over). Might make sense to either clarify that or switch it to mean the total number of languages.